spec/layers.md

```diff
@@ -0,0 +1,97 @@
+clear function
+==============
+**Status**: explainer.
+
+Provide a `beginLayer()` and `endLayer()` functions that create and close layers for Canvas.
+
+
+Rationale
+---------
+
+Currently, the only way to draw a series of drawing calls at the same time, while applying the same sort of effects (shadows, blur, alpha, filters, compositing) to all of the drawn objects, without having these effects being applied one to one to this object, is by using an auxiliary canvas and then, drawing that auxiliary canvas to the canvas.
+This proposal adds an alternative to that (the same way that Fullter in Google and Core Graphics at Apple do) by creating a layer that then will be drawn in one go when the layer ends.
+
+
+Proposal
+--------
+
+```webidl
+interface mixin CanvasState {
+  // extending original
+  undefined beginLayer();
+  undefined endLayer();
+};
+```
+The rendering state used for the layer rendering will use the current state of the canvas. The effects that will be used are:
```

Comment:
maybe rephrase this, as they are not "effects", and move it after the beginLayer. paragraph.